ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -0,0 +1,205 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import Qt.labs.settings 1.0
+
+import utils 1.0
+import shared 1.0
+import shared.panels 1.0
+import shared.popups 1.0
+import shared.status 1.0
+import shared.views.chat 1.0
+
+import StatusQ.Layout 0.1
+import StatusQ.Popups 0.1
+
+import "."
+import "../panels"
+import "../panels/communities"
+import "../popups"
+import "../helpers"
+import "../controls"
+import "../stores"
+
+StatusAppThreePanelLayout {
+    id: root
+
+    property var contactsStore
+    property bool hasAddedContacts: root.contactsStore.myContactsModel.count > 0
+
+    // Not Refactored
+    property var messageStore
+
+    property RootStore rootStore
+
+    property Component pinnedMessagesListPopupComponent
+    property var emojiPopup
+    property bool stickersLoaded: false
+
+    signal communityInfoButtonClicked()
+    signal communityManageButtonClicked()
+    signal profileButtonClicked()
+    signal openAppSearch()
+
+    Connections {
+        target: root.rootStore.stickersStore.stickersModule
+        onStickerPacksLoaded: {
+            root.stickersLoaded = true;
+        }
+    }
+
+    Connections {
+        target: root.rootStore.chatCommunitySectionModule
+        onActiveItemChanged: {
+            root.rootStore.openCreateChat = false;
+        }
+    }
+
+//    property var onActivated: function () {
+//        root.rootStore.chatsModelInst.channelView.restorePreviousActiveChannel();
+//        chatColumn.onActivated();
+//    }
```

Comment:
The `ChatView.qml` is a legacy `ChatLayout.qml`, just renamed. I didn't focus on removing anything from there.
Since you pointed it out I will remove it, I'm not a fan of dead, commented code.